src/products/index.ts

```diff
@@ -76,15 +75,35 @@ export type ProductVersions = {
      */
     getCurrencies(productUuid: string): Promise<ProductCurrency[]>;
   };
-};
+  [Version.V3]: {
+    /**
+     *  Retrieves a paginated list of all products
+     *
+     * Docs - https://docs.salable.app/api/v3#tag/Products/operation/getProducts
+     *
+     * @returns {Promise<GetAllProductsV3[]>} All products present on the account
```

Comment:
Should this be organisation instead of account? I don't believe we use account anywhere else and one user account could be in multiple orgs.